docker/hive2-ldap/files/conf/hive-site.xml

```diff
@@ -43,6 +22,16 @@
         <value>root</value>
     </property>
 
+    <property>
+        <name>hive.execution.engine</name>
+        <value>tez</value>
+    </property>
+
+<!--    <property>-->
+<!--        <name>hive.server2.enable.doAs</name>-->
+<!--        <value>false</value>-->
+<!--    </property>-->
```

Comment:
not sure if you meant to include this

Reply:
this feature can be useful, so I just commented it out ... it practically doesn't matter, since it's just a development environment.